### DISCUSSION TITLE
Can’t reach Vercel on home Network

### DISCUSSION THREAD

**Discussion by Olivier-Vromans** [vercel/community#2783] [Help] [answered]
### Summary

When I try to reach vercel.com from my home network I get an error that it can’t connect to the server but other websites work. So I try connecting via my phone thats also on that network but same issue. Only when I connect to my movile provider I can Reach the website. 

It happend right after I pushed a new change and had to get rid of some old env variables and try to add some new ones. After that I couldn’t reach the website.

### Steps to Reproduce

This is what I see on my phone:
![77D74970-6B3A-4367-A7EC-CE68B39D757B](https://github.com/vercel/community/assets/59826338/8d7feef2-04ef-4325-9ae8-feb2abd1123f)
![5C7E51AE-9CF3-4271-906B-AB3872EA9E22](https://github.com/vercel/community/assets/59826338/9288d2fe-38c5-470c-8058-b8de210b2fa0)
![C2D2AC9B-B91F-4783-9652-8BCBA2BD5424](https://github.com/vercel/community/assets/59826338/0b4e442b-69ff-4267-a1e4-2ad034166da4)
![7864BBAD-E636-469D-932D-8C8CEA815C75](https://github.com/vercel/community/assets/59826338/18465a6b-a48b-4597-822e-637267a210f8)

**Comment by Olivier-Vromans** [comment, marked as answer]
This morning when I woke up I tried again and it worked again, So maybe I got a timeout somehow

**Comment by serkan-bayram** [comment]
Having the same problem, can connect from some network and can't connect from some others since yesterday, it happened out of nowhere. Does anyone else have something like this?
